src/components/riddleDetail/_components/InfoLine.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Super nápad extrahovat komponenty InfoLine a InfoAccordion, asi bych se ale přiklonil k tomu je vynořit z podsložky components, protože jednak jsou to všechno stejně dílčí komponenty, jednak to je nekonzistentní se zbytkem projektu.